submission/utils.py

```diff
@@ -0,0 +1,96 @@
+import glob
+import json
+import os
+import shutil
+
+
+def get_new_d3m_path():
+    """
+    Gets the name of the newest version path for d3m/byudml
+    :return:
+    """
+    new_directory = max(glob.glob('submission/primitives/v????.?.?'))
+    byu_path = "byu-dml"
+    byu_dir = os.path.join(new_directory, byu_path)
+    return byu_dir
+
+
+def create_meta_script_seed(problem):
+    """
+    Update this as time goes on.  This is only for SEED datasets
```

Comment:
Use the docstring to describe what this method does. If there is a todo, mark it as a todo.